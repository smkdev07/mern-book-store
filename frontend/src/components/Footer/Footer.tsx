import React from 'react';

import { Container, Row, Col } from 'react-bootstrap';

const Footer: React.FunctionComponent = () => (
  <footer>
    <Container>
      <Row>
        <Col className="text-center py-3">Copyright &copy; Book Store</Col>
      </Row>
    </Container>
  </footer>
);

export default Footer;
