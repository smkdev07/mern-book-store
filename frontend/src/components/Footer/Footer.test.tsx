import React from 'react';
import { shallow } from 'enzyme';

import Footer from './Footer';

describe('<Footer />', () => {
  it('renders the <Footer /> component with a <footer> html tag', () => {
    const wrapper = shallow(<Footer />);
    const footer = wrapper.find('footer');
    expect(footer).toHaveLength(1);
  });

  it('renders the <Footer /> component', () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
