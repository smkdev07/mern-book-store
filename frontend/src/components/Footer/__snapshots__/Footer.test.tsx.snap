// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Footer /> renders the <Footer /> component 1`] = `ShallowWrapper {}`;
